go/store/val/tuple_descriptor.go

```diff
@@ -32,6 +32,7 @@ type TupleDesc struct {
 	Types []Type
 	cmp   TupleComparator
 	fast  fixedAccess
+	Addrs []int
```

Comment:
We could probably go without this field and calculate the number of ref'd fields from the `Types` field